src/adapter/auth-oso-rebac/src/oso_resource_service_impl.rs

```diff
@@ -53,79 +53,21 @@ impl OsoResourceServiceImplStateHolder {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-pub struct OsoResourceServiceImpl {
-    state_holder: Arc<OsoResourceServiceImplStateHolder>,
+pub struct OsoDatasetResourceServiceImpl {
     dataset_entry_svc: Arc<dyn DatasetEntryService>,
     rebac_service: Arc<dyn RebacService>,
-    account_service: Arc<dyn AccountService>,
 }
 
 #[component(pub)]
```

Comment:
Minor: 

Tip: if you use `#[dill::component]` over the structure declaration, you won't have to write `new()`

Reply:
No changes